src/session_impl.cpp

```diff
@@ -465,7 +464,7 @@ namespace aux {
 		// ---- auto-cap max connections ----
 		int max_files = max_open_files();
 		// deduct some margin for epoll/kqueue, log files,
-		// futexes, shared objects etc.
+		// mutexes, shared objects etc.
```

Comment:
actually, this was meant to say "futexes" (as in the linux kernel synchronization primitive)